doc/rule-descriptions.md

```diff
@@ -33,6 +33,8 @@
 | input-image-alt | Ensures &lt;input type=&quot;image&quot;&gt; elements have alternate text | cat.text-alternatives, wcag2a, wcag111, section508, section508.22.a | true |
 | label-title-only | Ensures that every form element is not solely labeled using the title or aria-describedby attributes | cat.forms, best-practice | true |
 | label | Ensures every form element has a label | cat.forms, wcag2a, wcag332, wcag131, section508, section508.22.n | true |
+| landmark-banner-is-top-level | A banner landmark identifies site-oriented content at the beginning of each page within a website | best-practice | true |
```

Comment:
It looks like most of the rule description read as descriptions of what the rule checks for, not as descriptions of the benefit of a given accessibility feature. We should be consistent. This would become something like, "Ensures a banner landmark is present at the top level of the document."

Reply:
Sorted. Weirdly enough the messages of these were quite different from the landmark-main-is-top-level. That one actually had a very nice description, so I just reused that.